vectorstores/chromadb/embedingfunc.go

```diff
@@ -0,0 +1,34 @@
+package chromadb
+
+import (
+	"context"
+
+	chroma "github.com/amikos-tech/chroma-go"
+	"github.com/tmc/langchaingo/embeddings"
+)
+
+var _ chroma.EmbeddingFunction = wrappedEmbeddingFunction{}
+
+// wrappedEmbeddingFunction is a wrapper around an embeddings.
+type wrappedEmbeddingFunction struct {
+	embeddings.Embedder
+}
+
+func (embedder wrappedEmbeddingFunction) CreateEmbedding(documents []string) ([][]float32, error) {
```

Comment:
One of my `TODO`s is to enable the injection of the embedder to use, so I'll see if this works for me.  However, I find it odd that you actually need to _run_ the embedder function and deal with its outputs.  I guess I'll find out more as I get to that...  Thanks for this example!

Reply:
cause chroma-go use `float32` but  `float64` in langchaingo

Reply:
This is one of my questions  needs to figure out with chroma-go owner.